LFS171x/docs/introduction-to-hyperledger-composer.md

```diff
@@ -0,0 +1,976 @@
+>
+> Let’s switch to the test tab to run some transactions.
+>
+> We start by creating a new Fisher participant, Alice. Notice the field validation working for the postal code.
+>
+> Now we can create Bob, our Restaurant owner. And Carla, our regulator.
+>
+> We can also create a Tuna fish asset. Notice the range validator for the tuna weight. Also note that we specify its owner to be Alice, the Fisher.
+>
+> Having our Participants and Assets, we can run a SellTuna transaction to transfer it to Bob. As you can see, the Tuna now has a new owner.
+>
+> Finally, we can browse all the transactions we have performed.
+
+## 10) Running the Composer REST Server
+We can also run the REST server to connect to the deployed business network and expose resources and transactions.
+
+```
+composer-rest-server -c admin@tuna-network -n never -w true
+```
+
+We can now access http://localhost:3000/ to explore the Composer REST API.
+
+> **VIDEO [EdX_4_REST.mp4]**
+> Transcript:
+
+> In this video we will learn about the Hyperledger Composer REST server.
+>
+> Let’s start the composer-rest-server. We specify our Tuna business network card and press enter to select the default options for the remaining arguments.
+>
+> We navigate to localhost:3000 in a web browser to access the REST server explorer. Here we can view the actions for a Participant, such as a Fisher. These include typical HTTP verbs such as GET, POST, PUT and DELETE.
+>
+> Let’s retrieve a list of Fisher objects. Naturally, we see Alice, who we created when using the Playground.
+>
+> We can do the very same thing with our Regulators. And with our Restaurant Owners. And with assets like Tuna.
+>
+> Importantly, we can also run Queries on the data. For instance, let’s find all the Tuna that is owned by Bob, our RestaurantOwner.
+>
+> Finally, the System tab contains API paths related to identity, the historian and allows us to ping the network. Let’s try it out.
+
+# Joining Hyperledger Composer Community
+
+## Learning more about Hyperledger Composer
+Hyperledger Composer project page provides links to code, documentation, examples and latests news:
+
+https://www.hyperledger.org/projects/composer
+
+The full documentation can be found here:
+
+https://hyperledger.github.io/composer/latest/
+
+It is detailed, well-organised and regularly updated, making it the first go-to resource when needing answers on Hyperledger Composer.
+
+## Community Meetings and Mailing List
+
+The community mailing list for Hyperledger Composer can be found here:
+
+https://lists.hyperledger.org/g/composer
+
+To find the next Hyperledger Composer meeting you can navigate to the Wiki page:
+
+https://github.com/hyperledger/composer/wiki/NextCommunityCall
+
+And the full list of meetings of Hyperledger projects is provided on this calendar:
```

Comment:
And the full list of meetings within the Hyperledger community is provided on this calendar: